lib/dbs/mysql.ts

```diff
@@ -91,3 +97,47 @@ export async function getStoreToken(storeHash: string) {
 export async function deleteStore({ store_hash: storeHash }: SessionProps) {
     await query('DELETE FROM stores WHERE storeHash = ?', storeHash);
 }
+
+// CHECKOUT Functions
+export async function setStorePlan(session: SessionProps) {
+    const { access_token: accessToken, context, plan, sub } = session;
+    // Only set on app install or subscription verification (load)
+    if ((!accessToken && !plan?.pid) || (plan && !plan.isPaidApp)) return null;
+
+    const contextString = context ?? sub;
+    const storeHash = contextString?.split('/')[1] || '';
+    const defaultEnd = Date.now() + (trialDays * 24 * 60 * 60 * 1000);
```

Comment:
this is exactly what we're doing for Firebase... should we be using sql's timestamp here and then convert later? or is this fine for now? @christensenep